docs/tutorials/setting-up-ethernet-over-usb-on-linux-command-line.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Setting Up Ethernet Over USB on Linux with Command Line
+subject: Ethernet Over USB
+---
+
+1.  On the EV3, first verify that the CDC driver is enabled. In brickman,
+    open the *USB* settings and make sure *CDC* is selected and active.
+
+    {% include screenshot.html source="/images/brickman/usb-cdc-active.png" %}
+
+2.  Then, in *Networking*, make sure *Gadget* is enabled.
+
+    {% include screenshot.html source="/images/brickman/networking-technologies-gadget-selected.png" %}
+
+3.  On your host computer, with the Ev3 connected via USB, open a *Terminal* and run
+
+        user@host:~$ ifconfig | grep usb
+  
+    Assuming there is only ```usb0``` the output should look like:
+  
+        usb0      Link encap:Ethernet  HWaddr 12:16:53:46:2e:42 
+  
+    Now set the ip address of ```usb0``` to ```10.42.0.1``` with the following command. The ```/24``` lets the host computer know the range of ip addresses connected to this interface.
+  
+        user@host:~$ sudo ip ad add 10.42.0.1/24 dev usb0
```

Comment:
No, actually it's not even persistent when the USB cable is removed and re inserted. 
It doesn't change `/etc/network/interfaces`.

I will find a way to do it persistently - edit /etc/network/interfaces is one, I'll check if there is another.